.github/workflows/autobuild.yaml

```diff
@@ -122,26 +122,48 @@ jobs:
         "-DCMAKE_CXX_FLAGS_RELEASE=${{ matrix.config.cflags }}"
         ${{ matrix.config.cmake }}
     - name: Ninja
-      run: ninja -C ${{ env.CLANGD_DIR }} clangd
-    - name: Archive
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd clangd-indexer
+    - name: Archive clangd
       run: >
         7z a clangd.zip
         ${{ env.CLANGD_DIR }}/LICENSE.TXT
-        ${{ env.CLANGD_DIR }}/bin/clangd*
+        ${{ env.CLANGD_DIR }}/bin/clangd
         ${{ env.CLANGD_DIR }}/lib/clang
-    - name: Upload artifact
+    - name: Archive indexing-tools
+      # FIXME: For now, indexing tools will be only available on Linux. Other
+      # platforms will be added soon.
+      if: matrix.config.name == 'linux'
+      run: >
+        7z a indexing-tools.zip
+        ${{ env.CLANGD_DIR }}/LICENSE.TXT
+        ${{ env.CLANGD_DIR }}/bin/clangd-indexer
+    - name: Upload clangd artifact
       uses: actions/upload-artifact@v1
       with:
-        name: ${{ matrix.config.name }}
+        name: ${{ matrix.config.name }}-clangd
```

Comment:
external tools match these names - I'd suggest leaving the clangd ones unchanged